etools-searchable-multiselection-menu.html

```diff
@@ -149,6 +149,11 @@
         color: rgba(0,0,0,0.54);
         background-color: #EEEEEE;
       }
+      .emptyValue {
```

Comment:
this is what the UX guys told me to do, on the fly because is not in the specification